test/create_git_history.sh

```diff
@@ -32,5 +32,20 @@ fakecommit d
 git checkout main
 fakecommit V
 fakecommit VI
+
+git checkout -b main-merge-target
```

Comment:
note to reviewer, I modified this file but i did not run it (since it would blow away the pre-existing commit history needed for prior tests).

Instead I ran:
```
cd test
sh setUp.sh
{{ all the commands I added }}
gomp main-merge-target main-merge-target~1 --cols 80 > output.txt
sh tearDown.sh
```